resources/[system]/scoreboard/ui/reportWebVitals.ts

```diff
@@ -0,0 +1,15 @@
+import { ReportHandler } from 'web-vitals';
+
+const reportWebVitals = (onPerfEntry?: ReportHandler) => {
```

Comment:
yea

Reply:
Yeah, we can honestly delete both this and the test file....